kubernetes/annotations_and_labels.md

```diff
@@ -34,6 +34,7 @@ This page defines common annotations and labels we set in Kubernetes objects.
   `giantswarm.io/organization=dev`.
 - `giantswarm.io/randomkey` - value should contain randomkey name as defined in
   github.com/giantswarm/randomkeys repo. This is used in encryption Secrets.
+- `giantswarm.io/reconciled-version` - value should contain version reconciled by the operator (formerly known as versionbundle version).
```

Comment:
How the operator is finding the resources its reconciling does not belong here, and this is not true for most of the current operators. I'd rather leave this out.

Reply:
It's not about how but what. This is very important piece of information IMO. We need make people aware that those two values are connected.

Reply:
To me `what` describe the purpose of this label: "version beeing reconciled"
This describe how because it tells how the operator is matching this version against the `giantswarm.io/reconciled-version` label on the CR.

I agree this is an important piece of information when you want to know how the operator works. I just don't feel this is the right place to document this here.

Reply:
Maybe it's not a perfect fit but I prefer to be practical.

Reply:
Proceed as you see the fit.

Reply:
> tells how the operator is matching this version against the giantswarm.io/reconciled-version label on the CR

For me this only tells how the system works. Operator can match the label in many different ways.